src/lm_buddy/paths.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+from typing import Annotated, Any
+
+from huggingface_hub.utils import HFValidationError, validate_repo_id
+from pydantic import BeforeValidator
+
+from lm_buddy.integrations.wandb import WandbArtifactConfig
+from lm_buddy.types import BaseLMBuddyConfig
+
+
+class FilesystemPath(BaseLMBuddyConfig):
+    """Absolute path to an object on the filesystem"""
```

Comment:
suggestion: "on the local Linux filesystem of the head node of Ray" or something more specific about where results are being processed

Reply:
This isn't always true because you may be running this code locally, in which case it simply corresponds to a path to load from disk.  I added the word "local" to clarify this.